docs/jupyter.md

```diff
@@ -32,7 +32,7 @@ First, if PlantCV is installed in the global Python search path, you can
 import the PlantCV library like normal:
 
 ```python
-import plantcv
```

Comment:
Is this true?

Reply:
Nope, should be revised to `from plantcv import plantcv` as you have it